### PR TITLE
Muhammad Usman - Feed API Challenge.

### PR DESCRIPTION
**Pull request by iTatla1**
Hi Caio and Mike,

Please review my PR; Please do let me know where I could improve. Thank You.

### CODE REVIEW

File: FeedAPIChallenge/Feed API/RemoteFeedLoader.swift [hunk 1]
```diff
@@ -5,20 +5,36 @@
...
+            switch result {
+            case let .success((data, response)):
+                do {
+                    let images = try FeedImageMapper.map(data: data, httpResponse: response)
+                    completion(.success(images))
+                }
+                catch _ {
+                    completion(.failure(Error.invalidData))
+                }
+            break
```
Comment (caiozullo) [inline FeedAPIChallenge/Feed API/RemoteFeedLoader.swift:33]: This `break` is unnecessary. You can remove it.

**Review by caiozullo — COMMENTED** [review]
Hi @iTatla1,

Congratulations, you've passed the challenge 🥇

All tests are passing, so your implementation complies with the specs. 

I only left a comment regarding an unnecessary `break` statement. Feel free to push changes.

Great job! ✅

File: FeedAPIChallenge/Feed API/RemoteFeedLoader.swift (hunk 1 above)
Comment (iTatla1) [reply FeedAPIChallenge/Feed API/RemoteFeedLoader.swift]: Resolved unnecessary Break. Thanks @caiozullo  😊

**Comment by caiozullo** [comment]
Nice one!

**Pull request closed without merging**